build/README.md

```diff
@@ -38,14 +38,16 @@ For example, the below config is used for running with two GPUs and FSDP for fin
     "per_device_train_batch_size": 4,
     "learning_rate": 1e-5,
     "response_template": "\n### Label:",
-    "dataset_text_field": "output"
+    "dataset_text_field": "output",
+    "lora_post_process_for_vllm": true
```

Comment:
Would be interested to hear if @Ssukriti has thoughts on this param but looks good to me and thanks for updating these docs!